LICENSE-binary

```diff
@@ -241,6 +241,7 @@ io.netty:netty-transport-native-unix-common
 io.netty:netty-transport-rxtx
 io.netty:netty-transport-sctp
 io.netty:netty-transport-udt
+io.netty:netty-handler-ssl-ocsp
```

Comment:
do we need the ssl? @FMX

Reply:
We do not need ssl now but the handle "ocsp" was introduced by Netty 4.1.86. I think we could exclude this dependency or accept this change. @pan3793

Reply:
I don't think the ssl work w/o other ssl jars, let's exclude it.

Reply:
ok, i will remove netty-handler-ssl-ocsp.